packages/shorebird_cli/test/src/auth/auth_test.dart

```diff
@@ -106,7 +106,7 @@ void main() {
         });
 
         test('returns AuthProvider.microsoft', () {
-          expect(jwt.authProvider, isA<MicrosoftAuthProvider>());
+          expect(jwt.authProvider, isA<MicrosoftAuthEndpoints>());
```

Comment:
```suggestion
          expect(jwt.authProvider, isA<MicrosoftAuthEndpoints>());
```

this reads a bit weird imo 🤷

Reply:
If the goal of this was to avoid naming conflicts with the shorebird auth provider concept couldn't we have a `ShorebirdAuthProvider` instead?

Reply:
Yep, missed renaming. Fixed